src/graphics/constants.js

```diff
@@ -1000,6 +1000,30 @@ export const TEXHINT_SHADOWMAP = 1;
 export const TEXHINT_ASSET = 2;
 export const TEXHINT_LIGHTMAP = 3;
 
+/**
+ * @constant
+ * @name pc.TEXTUREPROJECTION_CUBE
+ * @type {number}
+ * @description Texture data is stored in cubemap projection format.
+ */
+export const TEXTUREPROJECTION_CUBE = 0;
```

Comment:
could we make these 'cube', 'equirect' and 'octahedral' instead of 0, 1, 2?

Reply:
done, good idea.